src/index.js

```diff
@@ -0,0 +1,21 @@
+import create from "./lib/factory";
+
+export { create };
```

Comment:
cant we make something like `export {default as create} from "./lib/factory";` here?

Reply:
Nah we can't, there is a proposal but its early stages https://github.com/tc39/proposal-export-default-from so better to leave it :)